tools/rapids-twine

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# A utility script that wraps twine to support build tags (PEP427)
+#        
+# Positional Arguments:
+#   1) wheel dir
+set -exo pipefail
+export RAPIDS_SCRIPT_NAME="rapids-twine"
+
+rapids-require-env-var "TWINE_USERNAME"
+rapids-require-env-var "TWINE_PASSWORD"
+rapids-require-env-var "TWINE_REPOSITORY_URL"
```

Comment:
I removed `rapids-require-env` in lieu of `set -u` in the PR below

- https://github.com/rapidsai/gha-tools/pull/27

Reply:
based on the arguments in the docs below, you can probably get away with just adding the following `twine` flags with `set -u` to accomplish the same thing:

```sh
twine upload \
  --username "$TWINE_USERNAME" \
  --password "$TWINE_PASSWORD" \
  --repository-url "$TWINE_REPOSITORY_URL"
```


- https://twine.readthedocs.io/en/stable/#twine-upload